main.rb

```diff
@@ -0,0 +1,52 @@
+require "terminal-table"
+require_relative "planet"
+require_relative "solar_system"
+
+def main
+  solar_system = SolarSystem.new("A Wrinkle in Time")
+
+    camazotz = Planet.new(“Camazotz”, “red”, 68.9, 22.777, “Ruled by a disembodied brain”)
+    ixchel = Planet.new(“Ixchel”, “pink”, 49, 10.8E9, “Inhabited by sightless creatures”)
+    uriel = Planet.new(“Uriel”, “blue”, 10, 411.5E1, “Lots of extremely tall mountains”)
+
+    solar_system.add_planet(camazotz)
+    solar_system.add_planet(ixchel)
+    solar_system.add_planet(uriel)
+
+  directions = ""
+
+  until directions == "exit"
+    puts "Please enter list planets to see all the planets, planet details to learn more about each planet, or add planet to enter a new planet.
+    directions = gets.chomp.downcase
+    if directions == "list planets"EXIT
```

Comment:
More syntax errors: missing and end-quote on line 19, and an extra `EXIT` at the end of line 21.